pylint/test/functional/invalid_envvar_value.txt

```diff
@@ -0,0 +1,5 @@
+import os
```

Comment:
Take a look at the other test files from the `functional` directory. The `.txt` should be a mapping of the error occurrences from the `.py` file.